internal-packages/next-dev/package.json

```diff
@@ -0,0 +1,35 @@
+{
+	"name": "@cloudflare/next-on-pages-next-dev",
+	"version": "0.0.1",
+	"main": "dist/index.cjs",
+	"scripts": {
+		"lint": "eslint src",
+		"types-check": "tsc --noEmit",
+		"build:js": "esbuild --bundle --format=cjs ./src/index.ts --external:miniflare --outfile=./dist/index.cjs --platform=node",
+		"build:types": "tsc --emitDeclarationOnly --declaration --outDir ./dist",
+		"build:js:watch": "npm run build:js -- --watch=forever",
+		"build:types:watch": "npm run build:types -- --watch",
+		"build": "npm run build:js && npm run build:types",
+		"build:watch": "npm run build:js:watch & npm run build:types:watch",
+		"test": "npx vitest --config vitest.config.ts"
+	},
+	"files": [
+		"dist",
+		"dev-init.cjs",
+		"dev-init.d.ts",
+		"devBindingsOptions.ts"
+	],
+	"dependencies": {
+		"miniflare": "^3.20231002.0",
+		"node-fetch": "^3.3.2"
```

Comment:
Not undici?

Reply:
Or built in fetch?

Reply:
we're still requiring node 16.... although since it's deprecated we could switch to 18... 🤔 

Regarding `node-fetch` or `undici`, I just happened to use `node-fetch` if undici is preferred I can totally switch over that 🙂

Reply:
mh... seems like using undici is bloating the output? 🤔 

currently this is what we get (with `node-fetch`):
![Screenshot 2023-11-02 at 23 03 55](https://github.com/cloudflare/next-on-pages/assets/61631103/bf1a199e-d81e-4d1e-aeb2-bc0eb8d8d212)

and this is what we get if I try using `undici`:
![Screenshot 2023-11-02 at 23 04 14](https://github.com/cloudflare/next-on-pages/assets/61631103/c56e2ed4-7155-421d-9434-7b1468a5d5eb)

I'm tempted to then just leave `node-fetch` for now and as soon as we can we just switch to node 18 and use the built in one (but we can only do that in a major release 😓)

what do you think?